azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-config-topics.adoc

```diff
@@ -0,0 +1,9 @@
+= Azure Service Bus Management 1.0 Additional Configuration Topics - Mule 4
+
+Anypoint Azure Service Management connector cannot perform a connectivity test at startup using the `Test Connection` configuration option when you restrict access to your resources
+and you have a security policy with permissions at the resource level only. This scenario occurs because the security policy targets the root level of the namespace, which might be forbidden due to the customized policy applied to the shared access key.
```

Comment:
For future update if devs can provide additional information: it would be useful to have an example of how the `security policy` is targeting the root level of the namespace, and also the `customized policy` applied to the shared access key. 
This topic talks about additional configuration but does not mention what type of configuration. With `Test Connection` configuration` do you refer to the button in the Global Elements window?